builds/e2e/templates/e2e-setup.yaml

```diff
@@ -161,14 +161,15 @@ steps:
     if ($IsWindows)
     {
       $context['installerPath'] = Convert-Path '$(Build.SourcesDirectory)/scripts/windows/setup'
+      $context['testTimeoutMinutes'] = 8
```

Comment:
I'm just curious -- what was the value before your change when it wasn't explicitly set in this file?

Reply:
The default is 5 minutes. See https://github.com/Azure/iotedge/blob/release/1.1/test/Microsoft.Azure.Devices.Edge.Test/helpers/Context.cs#L108.